contracts/managers/crowdfunding-softcap/CfManagerSoftcapFactory.sol

```diff
@@ -9,7 +9,7 @@ import "../../registry/INameRegistry.sol";
 contract CfManagerSoftcapFactory is ICfManagerSoftcapFactory {
     
     string constant public FLAVOR = "CfManagerSoftcapV1";
-    string constant public VERSION = "1.0.21";
+    string constant public VERSION = "1.0.23";
```

Comment:
When I update version on any of the versioned files , I will set it to the latest one defined in package.json . For example, one factory might remain unchanged while the app updates 2 or 3 times , and then when this factory gets changed it will take the latest version of the *project* (not its internal version). This way we can track updates more easily I'd say